Lombiq.Tests.UI/Services/OrchardCoreInstance.cs

```diff
@@ -238,14 +245,18 @@ private async Task StartOrchardAppAsync()
 
         await _command.ExecuteDotNetApplicationAsync(
             stdErr =>
+            {
+                var dotnet = "dotnet" + ExecutableExtension;
+                var note = stdErr.Text.ContainsOrdinalIgnoreCase("Failed to bind to address")
+                    ? " This can happen when there are leftover " + dotnet +
+                      " processes after an aborted test run or some other app is listening on the same port too."
+                    : string.Empty;
```

Comment:
Some note about what might be an issue or where to look?

Reply:
Look, I just moved the text out of the exception constructor to make it more readable. [Ask the person who wrote it originally](https://github.com/Lombiq/UI-Testing-Toolbox/commit/5acd4781f8484cc99d631f515dc96c620ca97738), I hear you are on good terms. 😁

Reply:
To paraphrase Knuth, beware of bugs in the above code; I have only written it, not reviewed it. :D